sample/java/src/main/java/com/datadog/android/sample/SampleApplication.java

```diff
@@ -143,4 +175,17 @@ public Logger getLogger() {
     public static SampleApplication fromContext(Context context) {
         return (SampleApplication) context.getApplicationContext();
     }
+
+    private static final class CustomViewEventMapper implements ViewEventMapper {
+        private final SecureRandom random = new SecureRandom();
+
+        @Override
+        public ViewEvent map(@NonNull ViewEvent event) {
```

Comment:
What's the purpose of this mapper? I could understand such a mapper in a test or a mock but in this case I don't get what use case it's trying to solve?

Reply:
It was for me to make sure that the events are handled correctly if we return null from Java. As it doesn't harm the application functionality (the original event will still be returned) I thought to leave it here but if you think is not needed will remove it.